make-release.sh

```
@@ -197,7 +197,7 @@ createPRToMainBranch() {
   resetChanges main
   local tmpBranch="copy-${RELEASE_VERSION}-bundle-to-main"
   git checkout -B "${tmpBranch}"
-  git diff refs/heads/main...refs/heads/${RELEASE_BRANCH} | git apply -3
+  git diff refs/heads/main...refs/heads/${RELEASE_BRANCH} ':(exclude)pkg/defaults/defaults.go' | git apply -3
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)